svg-2-android-vector/src/main/java/com/quittle/svg2androidvector/Svg2AndroidVectorPlugin.java

```diff
@@ -53,14 +56,25 @@ public void apply(final Project project) {
     }
 
     private static void onAfterEvaluate(final Project project, final Svg2AndroidVectorExtension extension) {
+        // The base Android extension, where all resources can be found.
+        final BaseExtension androidExtension = project.getExtensions().findByType(BaseExtension.class);
+        if (androidExtension == null) {
+            // This might mean it was applied to a non-Android project, possibly just the root project.
+            return;
+        }
         final TaskContainer taskContainer = project.getTasks();
         final Task parentTask = taskContainer.create(CONVERSION_PARENT_TASK_NAME);
         // An early Android task all the conversion tasks should be a dependency of
-        taskContainer.findByName(EARLY_ANDROID_TASK_NAME).dependsOn(parentTask);
+        final AppExtension androidApp = project.getExtensions().findByType(AppExtension.class);
+        final LibraryExtension androidLib = project.getExtensions().findByType(LibraryExtension.class);
```

Comment:
Don't forget about ` TestExtension`

Reply:
Oh, I had never used that so I didn't realize it was a thing to handle.

Reply:
Can't actually figure out how to get one of these - I can get TestVariants from the app and lib extensions, but can't get a test extension.